Added non-food processor alternative for crafting Soy Milk

Myfharad opened the pull request:
#### Summary
Content "Added non-food processor alternative for crafting Soy Milk"

#### Purpose of change

Soy Milk's only recipe locked its crafting behind a Food Processor, and well Soy Milk isn't a technologically dependent drink as it was originated in ancient China.

#### Describe the solution

Mimicked the Almond Milk recipe that involved using qualities for tools instead of the Food Processor tool and made a Soy Milk version of it.

#### Describe alternatives you've considered

Travelling to China to ask how they got the food processors during the [Eastern Han Dynasty](https://en.wikipedia.org/wiki/Soy_milk#History)

#### Testing

It was just a recipe copy paste and edit, if anything is missing or forgotten it should let me know

#### Additional context

If anything is missing just let me know

*whispers* I just wanted to make Tofu but didn't want to look for a Food Processor hehe

Myfharad committed:
Update drinks.json

Added Soy Milk recipe that almost completely mimics Almond Milk's recipe tool requirements for Non-Food Processor recipe

Myfharad committed:
Update drinks.json

Added a ID_Suffix, though no idea if it needs to be a completely unique suffix, but lowkey we'll see if the build fails lol

Maleclypse commented:
https://www.marystestkitchen.com/diy-soy-milk-recipe/  compare this to people discussing mortar and pestle vs blender and mortar and pestle is double the length of time.  And I feel like maybe all the recipes said theres a heat component but i could be wrong on that

Myfharad committed:
Update drinks.json

No idea how I removed the heating "tool", and slightly more than doubled the time of completion for the Soy Milk using a traditional mortar
